index.html

```diff
@@ -54,68 +54,47 @@
       <h1>
         Device and Sensors Working Group
       </h1>
-    </div><!--
-                      <div id="navigation">
-                          <ul>
-                          </ul>
-                      </div>
-                  -->
+    </div>
     <div id="main">
-      <!--p class="logo"><a href="/"><img alt="W3C" src="/Icons/w3c_home"/></a></p-->
-      <a class="logo" href="http://www.w3.org/"><img alt="W3C" height="48" src=
-      "https://www.w3.org/StyleSheets/TR/2016/logos/W3C" width="72"></a>
+      <a class="logo" href="https://www.w3.org/"><img alt="W3C" height="48"
+      src="https://www.w3.org/StyleSheets/TR/2016/logos/W3C" width="72"></a>
       <dl>
         <dt>
           On this page:
         </dt>
         <dd>
           <a href="#mission">Mission</a> | <a href="#roadmap">Roadmap</a> |
-          <a href="#future">Future Work</a> | <a href="#inputs">Submissions</a>
-          | <a href="#participate">Participate</a> | <a href=
-          "#meetings">Meetings</a> | <a href="minutes">Minutes</a> | <a href=
-          "#comm">Communications</a> | <a href="#docs">Documents</a> | <a href=
-          "#history">History</a>
+          <a href="#participate">Participate</a> | <a href=
+          "#meetings">Meetings</a> | <a href="#comm">Communications</a> |
+          <a href="#ipr">Patent Policy</a> | <a href="history.html">History</a>
         </dd>
         <dt>
           Nearby:
         </dt>
         <dd>
-          <a href=
-          "http://www.w3.org/2009/dap/wiki/ImplementationStatus">Information on
-          Implementations</a> | <a href=
-          "http://www.w3.org/2009/dap/wiki/Main_Page#Work_Documentation">WG
-          Notes and Checklists</a> | <a href=
-          "http://www.w3.org/2009/dap/wiki/Main_Page">WG wiki</a> | <a href=
-          "http://www.w3.org/2008/webapps/wiki/Main_Page">WebApps WG</a> |
-          <a href="http://www.w3.org/html/wg/">HTML WG</a>
-        </dd>
-        <dt>
-          Related:
-        </dt>
-        <dd>
-          <a href="http://www.w3.org/Mobile/mobile-web-app-state/">Standards
-          for Web Applications on Mobile: Current State and Roadmap</a>
+          <a href="#mediacapture">Media Capture Task Force</a> | <a href=
+          "https://www.w3.org/2009/dap/wiki/ImplementationStatus">Implementation
+          Status</a> | <a href="https://www.w3.org/2009/dap/wiki/Main_Page">WG
+          wiki</a> | <a href="https://www.w3.org/WebPlatform/WG/">Web Platform
+          WG</a> | <a href="https://www.w3.org/html/wg/">HTML WG</a>
```

Comment:
as long as we're updating this, I don't think the HTML WG link should stay (since that's now the HTML Media extensions WG)